Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -32,7 +32,7 @@
       dead: corgi_dead
   - type: Butcherable
     spawned:
-    - id: FoodMeatCorgi
+    - id: FoodMeat
       amount: 2
```

Comment:
other corgis giving corgi meat is fine, doesnt have to just be ian

Reply:
the thing is theyre dropping the traitor objective currently, so cargo just orders corgis and gets infinite traitor objectives

Reply:
this is something i view as unhealthy for the game as ive seen it happening very often